mllam_data_prep/create_dataset.py

```diff
@@ -127,6 +135,10 @@ def create_dataset(config: Config):
             dataset_name=dataset_name,
         )
 
+        # TODO: remove this once grid_mapping is set in config file
+        for var in ds.data_vars:
+            ds[var].attrs["grid_mapping"] = "crs"
+
```

Comment:
@leifdenby this sets the `grid_mapping` attribute for now here and so we can postpone the discussion on the structure of the `variables` section in the config file